### PR TITLE
Add react ui to project

### PR DESCRIPTION
**Pull request by slaroche**

### CODE REVIEW

**Comment by aviau** [comment]
Can you rebase?

File: ui/.gitignore [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+.pnp.js
+
+# testing
+/coverage
+
+# production
+/build
+
+# misc
+.DS_Store
```
Comment (aviau) [inline ui/.gitignore]: I am not fond of 100-lines gitignores.

Can we just keep it to the thing we actually need?

File: ui/README.md [hunk 2]
```diff
@@ -0,0 +1,46 @@
+# Getting Started with Create React App
```
Comment (aviau) [inline ui/README.md]: Not sure we need to commit this?

This looks like standard copypasta.

We can only add the parts we need to the root readme.

File: ui/.gitignore (hunk 1 above)
Comment (slaroche) [reply ui/.gitignore]: yes I wiil keep only what I need.

File: ui/README.md (hunk 2 above)
Comment (slaroche) [reply ui/README.md]: this is auto genated

**Pull request merged**